overlay/backend/services/generate-delta-worker/tests/docker-compose.yml

```
@@ -0,0 +1,15 @@
+name: acceptance
+services:
+  generate-delta-worker:
+    build:
+      args:
+        - BIN_FILE=backend/tests/bin/generae-delta-worker.test
```

Since the context is in this directory, the build context must be updated to make the paths resolve.
```suggestion
    build:
      args:
        - BIN_FILE=backend/tests/bin/generae-delta-worker.test
      context: ../../../..
```